api/meetups/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+from users.models import User
+import uuid
+import datetime
+"""
+Models for the meetups
```

Comment:
Should the docstring be at the top of the file?

Reply:
@Ogutu-Brian

Reply:
It should be at the top, I have fixed that, thank you @Bakley